acceptance/setup/pre_suite/30_generate_ssl_certs.rb

```diff
@@ -1,7 +1,14 @@
 step "Run an agent to create the SSL certs" do
+  if options[:type] == 'aio' then
+    hosts.each do |host|
+      on(host, puppet("agent -t"), :acceptable_exit_codes => [0,1])
+    end
+    on master, puppet("cert --sign --all"), :acceptable_exit_codes => [0,24]
```

Comment:
@kbarber you are correct. The autosign feature performs this duty. This code is not needed. I will remove it.